src/VisualStudio/CSharp/Test/GlyphExtensionsTests.cs

```diff
@@ -169,6 +170,14 @@ public void TestWithEventsMemberGlyph()
                 isWithEvents: true);
         }
 
+        [Fact, WorkItem(26642, "https://github.com/dotnet/roslyn/issues/26642")]
+        public void TestNoReferenceToImageCatalog()
```

Comment:
I should also say: I encouraged @JieCarolHu to write this test no matter the method taken to achieve this fix;, that way nobody innocently adds the assembly reference back and breaks it again.